src/posts/xwrf/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Making WRF output data pythonic using xWRF'
+date: '2022-09-20'
+authors:
+  - name: Jon Thielen
+    github: jthielen
+  - name: Lukas Pilz
+    github: lpilz
+---
+
+![xWRF logo](https://github.com/xarray-contrib/xwrf/blob/main/docs/source/_static/xwrf_logo_bg_light.svg)
+
+In the past, when analyzing [Weather Research and Forecasting (`WRF`)](https://www.mmm.ucar.edu/weather-research-and-forecasting-model) data with python you would be restricted to using the [`wrf-python`](https://wrf-python.readthedocs.io/en/latest/) package. This is not very convenient, especially when working on very large datasets since it uses compiled Fortran code in the background essentially preventing the user from scaling to cluster-scale using utilities like [`dask`](https://www.dask.org/).
+
+Here, we present to you [`xWRF`](https://github.com/xarray-contrib/xwrf/), a lightweight interface for working with the model output in [Xarray](https://docs.xarray.dev/en/stable/). Using `xWRF` enables a seamless integration of the unique `WRF` data format into Xarray and the [Pangeo](https://pangeo.io/) software stack. It achieves this by:
+
+1. transforming `WRF` data into CF- and COMODO-compliant xarray datasets
+2. converting `WRF` units into [`pint`](https://pint.readthedocs.io/en/stable/)-friendly ones
+3. generating projection coordinates and providing a [`pyproj.CRS`](https://pyproj4.github.io/pyproj/dev/api/crs/crs.html) object
+
+In this post, we will show how `xWRF` works together with other utilities in order to post-process a large `WRF` output dataset.
+
++++
+
+## Installing `xWRF`
+
+Before we start using `xWRF`, we need to install it. We can do this by following these steps!
+
+1. Install this in your python environment using pip (`pip install xwrf`)
+2. Open up a notebook and use the imports shown below!
+
+```{code-cell} ipython3
+import holoviews as hv
+import hvplot
+import hvplot.xarray
+import xarray as xr
+import xwrf
+from distributed import Client
+from dask_jobqueue import SLURMCluster
+import pint_xarray
+import numpy as np
+
+hv.extension('bokeh')
+```
+
+## Spin up a Cluster
+
+```{code-cell} ipython3
+cluster = SLURMCluster(name='dask-cluster',
+                      cores=128,
+                      memory='125G',
+                      processes=11,
+                      interface='ib0',
+                      queue='compute',
+                      walltime='8:00:00',
+                      asynchronous=0)
+n_workers = 2
+cluster.scale(n_workers)
+client = Client(cluster)
+client.wait_for_workers(n_workers=n_workers)
+client
+```
+
+## Load the WRF output
```

Comment:
A brief mention of intake would be helpful here, as well as an overview of what this dataset is... why are you using it in this tutorial

Reply:
Added some info. Please check if the level of detail is okay. Also, I'm not sure if the analysis in and of itself is sensible (from a physical/GCM standpoint) - it was simply the first thing I could think of which showed off all the nice `xWRF` features.